### DISCUSSION TITLE
Application to Github Education rejected for 'full name as appears in your academic affiliation'

### DISCUSSION THREAD

**Discussion by DarkStrik3** [community/community#57720] [GitHub Education]
### Hi everyone,

So basically, I have applied twice and have been rejected. I do not exactly understand how to confirm my full name, as I don't want to edit my username to my full name to preserve anonymity online. I have submitted my school ID which includes a valid date, name, photo, and DoB. It may be a bit blurry since my laptop camera isn't the best. What can I do to resolve this issue? 

-You are unlikely to be verified until you have completed your [GitHub user profile] with your full name as it appears in your academic affiliation document plus a short bio.

**Comment by farhannorth** [comment]
Your application cannot be reviewed until you fix the following:
Your uploaded proof has already been used previously.

**Reply by DarkStrik3** [reply]
What does it mean by the name bit though?

**Comment by farhannorth** [comment]
Your application cannot be reviewed until you fix the following:
You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.
Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

**Reply by DarkStrik3** [reply]
That is exactly the same as what the bot said. Can you please explain what that means? I have used the device camera (which you would know had you read the entire post) the first time, and the second time I tried to upload my school timetable.

**Comment by samus-aran** [comment]
Hi @DarkStrik3 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Hope this helps 👍 Let us know how you get on
